.github/workflows/smoke-test/smoke-test.sh

```diff
@@ -88,6 +88,17 @@ install_darwin() {
     cp -a "$srcApp" "$(dirname "$destApp")"
     xattr -d -r -s -v com.apple.quarantine "$destApp"
 
+    # Check that the image is compressed
+    local compressionRatio
+    compressionRatio="$(hdiutil imageinfo "$archiveName" \
```

Comment:
Seems to be missing the `-plist` option:

```suggestion
    compressionRatio="$(hdiutil imageinfo -plist "$archiveName" \
```

Reply:
Thanks, that was indeed a bug. Now I'm wondering why if I tested it incorrectly…